### PR TITLE
[Unity] Update docs for operators

### PR DESCRIPTION
**Pull request by Ubospica**
This PR:
- updates docs for several operators,
- add an `1` default value for the `axis` parameter of batch_norm,
- rename module `tvm.relax.transform.legalize_ops.creation` to `tvm.relax.transform.legalize_ops.create`, which is align with `tvm.relax.op.create`, and
- fixes a previous upstream error in tests/python/relax/test_transform_legalize_ops_grad.py

### CODE REVIEW

**Review by yongwww — APPROVED** [review]

File: python/tvm/relax/op/nn/nn.py [hunk 1]
```diff
@@ -659,14 +659,15 @@ def batch_norm(
     beta: Expr,
     moving_mean: Expr,
     moving_var: Expr,
-    axis: int,
+    axis: int = 1,
```
Comment (Hzfengsy) [inline python/tvm/relax/op/nn/nn.py:662]: Is there any reason to add such default value?

Comment (Ubospica) [reply python/tvm/relax/op/nn/nn.py:662]: Common batch norm operators takes the axis indexed 1 as the axis to calculate mean and variance, such as torch.nn.functional.batch_norm.

Comment (Ubospica) [reply python/tvm/relax/op/nn/nn.py:662]: Our difference from torch is that we may have layouts like NHWC, but torch only accepts NCHW. Considering this maybe we can remove the default value 1.
